src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -7406,6 +7406,25 @@ components:
           items:
             type: string
 
+    SnapshotBuilderTable:
```

Comment:
Is the idea that this would eventually be added to `SnapshotBuilderSettings`?

Reply:
That's correct

Reply:
I've detailed the future plan in this ticket: https://broadworkbench.atlassian.net/browse/DC-1120